src/screens/Home/components/PaymentButton.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
```

Comment:
Can `OutlinedCardList` be used here instead of creating another component? Looks like it could potentially work